### DISCUSSION TITLE
Zigbee2mqtt 1.25.0 compatibility

### DISCUSSION THREAD

**Discussion by Ivano62** [Koenkk/zigbee2mqtt#12032] [General]
Hi, I have read the description of the latest release. It is not clear to me if the transition to the Matter protocol maintains compatibility with the RaspBee II coordinator and all the zigbee devices (Xiaomi, Legrand, Tuya, etc.) that I currently use.
Thanks for now for clarification on the matter

**Comment by Ivano62** [comment]
Can anyone help me?

**Reply by lux73** [reply]
Come on Dude...   Yesterday it was 1. April ;-)

![april-april-0001](https://user-images.githubusercontent.com/3129990/161371997-4e0fe2e8-598b-472b-a47a-0f4b1721a749.gif)
